docs/maintainer-build-platforms.md

```diff
@@ -0,0 +1,78 @@
+# Build Platforms
+
+This document describes how to install and maintain a LinuxKit development platform. It will grow over time.
+
+The LinuxKit team also maintains several Linux-based build platforms. These are donated by Equinix Metal (arm64) and IBM (s390x).
+
+## Platform-Specific Installation
+
+### arm64 and amd64
+
+The `amd64` and `arm64` platforms are fully supported by most OS vendors and Docker. Just upgrade to the latest OS and install the latest Docker using the
+packaging tools. As of this writing, that is Ubuntu/Debian with `apt`, or RHEL/CentOS/Fedora with `yum`.
+
+Docker does not recommend that you using the packages released by the OS vendors, as those tend to be out of date. Follow the instructions
+[from Docker](https://docs.docker.com/engine/install/).
+
+### s390x
+
+The s390x has modern versions of most OSes, including RHEL and Ubuntu, but does not have recent versions of docker, neither as
+`apt` packages for Ubuntu, nor as static downloads. In any case, these static downloads mostly are replicas.
```

Comment:
Updated packages should be available soon (it's being worked on with IBM)

Reply:
You mean all of that work was for nothing? :-)

Not really; it got me going today. Let's keep this as is, and we can update it when the updated packages are ready.

Reply:
(hides) _hopefully_ soon the work becomes redundant yes. Until then 👍 👍  (thanks!). Your PR just caught my eye, and I thought I'd give you a heads-up.

Reply:
Definitely happy for it!